Acquisition/Interface/source/Lock.cpp

```diff
@@ -49,11 +50,16 @@ Lock::Lock(string name) {
     } else {
         pid_t pid;
 
-        fscanf(lockFile, "%10d", &pid);
+        ///@TODO - We need to update this so that we use strtol instead of fscanf to convert string to int
```

Comment:
This does more than convert a string to to an int. The lock file is physically written with the process id so that it can be traced when something fails. If that process is still running the user knows someone else may have left the acq open, and if it is dead they know they can remove the lock file safely. In any case this lock really shoulc just be changed to an atomic lock.